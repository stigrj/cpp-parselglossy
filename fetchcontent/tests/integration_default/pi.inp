title = "This program computes Pi"
method = integration
