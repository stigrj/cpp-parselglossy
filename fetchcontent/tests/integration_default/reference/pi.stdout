
-------------- JSON input -----------------
{
  "Darts": {
    "n_darts": 1000,
    "random_seed": 0
  },
  "Integration": {
    "n_steps": 1000
  },
  "method": "integration",
  "title": "This program computes Pi"
}
-------------------------------------------

-------------- JSON output ----------------
{
  "method": "Numerical integration",
  "pi": 3.1415927369231307,
  "title": "This program computes Pi"
}
-------------------------------------------

